lib/es6/caml_exceptions.js

```diff
@@ -14,11 +14,11 @@ function create(str) {
   return str;
 }
 
-function is_extension(e) {
-  if (e == null) {
-    return false;
+function is_extension(any) {
+  if (any) {
```

Comment:
Can this be made more precise? Should it be not `null`, not `undefined`, both?
Perhaps it's safe anyway. Just double checking.

Reply:
It's safe, but I wonder why `&&` was generated to `if/else`